environments/stg/backend_lb_config.yaml

```diff
@@ -33,6 +33,8 @@ gateways:
         component: case-documents-api
       - product: ia
         component: aip-frontend
+      - product: ia
```

Comment:
This is not needed, i cannot think why this is added.

Reply:
Wouldn't be better to have by default? Frontends will talk to other backends in most cases, won't them?